.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/ChineseDatePeriodParserConfiguration.cs

```diff
@@ -869,9 +870,18 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
 
             var pr1 = this.config.DateParser.Parse(er[0], referenceDate);
             var pr2 = this.config.DateParser.Parse(er[1], referenceDate);
-            if (pr1.Value == null || pr2.Value == null)
+
+            if (er.Count >= 2)
             {
-                return ret;
+                var dateContext = BaseDatePeriodParser.GetYearContext(config.DatePeriodYearRegex, config.RelativeRegex, new ChineseDateExtractorConfiguration(), er[0].Text, er[1].Text, text);

```

Comment:
No need to creates a new instance of `ChineseDateExtractorConfiguration`, just use `this.SingleDateExtractor`.